kernels/quantized/CMakeLists.txt

```diff
@@ -53,11 +55,18 @@ message("Generated files ${gen_command_sources}")
 # dependency of the other(s). This is not allowed by the Xcode "new build
 # system".
 if(NOT CMAKE_GENERATOR STREQUAL "Xcode" AND EXECUTORCH_BUILD_QUANTIZED_OPS_AOT)
+  gen_selected_ops(LIB_NAME "quantized_aot_ops_lib" OPS_SCHEMA_YAML
```

Comment:
I see 2 instances of quantized_aot_ops_lib which i believe should be quantized_ops_aot_lib (swap ops and aot in the string).

I don't think this code path is tested properly in CI at the moment as this change meant the .so was empty and so no quant ops were registered on the xnnpack test script.

Reply:
Thanks for catching it.